zerver/tests/test_import_export.py

```diff
@@ -428,3 +442,104 @@ def get_set(data: List[Dict[str, Any]], field: str) -> Set[str]:
 
         exported_messages_recipient = get_set(messages['zerver_message'], 'recipient')
         self.assertIn(list(exported_messages_recipient)[0], exported_recipient_id)
+
+    """
+    Start tests for import
+    """
+    def test_import_realm(self) -> None:
+
+        realm = Realm.objects.get(string_id='zulip')
+        RealmEmoji.objects.get(realm=realm).delete()
+        self._export_realm(realm)
+
+        with patch('logging.info'):
+            do_import_realm('var/test-export', 'test-zulip')
+
+        # sanity checks
+
+        # test realm
+        self.assertTrue(Realm.objects.filter(string_id='test-zulip').exists())
+        imported_realm = Realm.objects.get(string_id='test-zulip')
+        self.assertNotEqual(imported_realm.id, realm.id)
+
+        # test users
+        realm_user_emails = set([
```

Comment:
These may be more compact and simpler with set comprehensions, so fitting on one line and improving readability?